README.md

```diff
@@ -132,7 +132,7 @@ Currently we support the following puppet providers:
 | Protocol | Puppet Provider | Environment Variable |
 | --- | --- | --- |
 | Web | PuppetPuppeteer | `export WECHATY_PUPPET=wechaty-puppet-puppeteer` |
-| iPad | PuppetPadchat | `export WECHATY_PUPPET=wechaty-puppet-padchat` |
+| iPad | PuppetPadchat | `export WECHATY_PUPPET=wechaty-puppet-padpro` |
```

Comment:
Please change PuppetPadchat to PuppetPadpro as well. :smile:

Reply:
Yes, of course.